proposals/kubeflow-steering-committee-election-proposal.md

```diff
@@ -0,0 +1,125 @@
+# KSC Election Process Proposal
+
+
+## 2 cohorts and Staggered elections
+
+KSC consists of 5 members, made up of 2 cohorts of 3 members and 2 members, respectively. Each cohort will be elected every 2 years, with the election schedule staggered so elections for the 2 rotations occur on alternate years. The staggered cohorts are designed to provide continuity and stability in the project leadership of Kubeflow.  
+The only exception to the above rule is that the first 2-member rotation consists of 2 members from the Interim KSC, and will be re-elected after 1 year. The goal of having interim KSC members participate in the first KSC is to provide guidance, facilitate asset transitions from Google, and hopefully make the transition in open governance a smooth process.  
+The table below illustrates the cohorts and election schedules:
+
+<table>
+  <thead>
+    <tr>
+      <th>Timeline</th>
+      <th>2-member cohort</th>
+      <th>3-member cohort</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>2023</td>
+      <td>Interim KSC member A<br>
+Interim KSC member B</td>
+      <td>New election</td>
+    </tr>
+    <tr>
+      <td>2024</td>
+      <td>New election</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>2025</td>
+      <td></td>
+      <td>New election</td>
+    </tr>
+    <tr>
+      <td>2026</td>
+      <td>New election</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>2027</td>
+      <td></td>
+      <td>New election</td>
+    </tr>
+    <tr>
+      <td>…</td>
+      <td></td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+
+## Election Procedure
+
+### Timeline ###  
+Steering Committee elections are held annually. 4 weeks or more before the election, the Steering Committee will appoint Election Officer(s) (see below). 2 weeks or more before the election, the Election Officer(s) will issue a call for nominations, publish the list of eligible voters, and open the call for exceptions. One week before the election the call for nominations and exceptions will be closed. The election will be open for voting not less than two weeks and not more than four. The results of the election will be announced within one week of closing the election. New Steering Committee members will take office in December of each year on the date the results are announced.
```

Comment:
+1, if not a specific date, I think it's worth mentioning that it should start in Sep/Oct every year.

Reply:
I am open to suggestions. Curious what other people might suggest.

Reply:
I think something similar to [Kuberenetes steering election schedule](https://github.com/kubernetes/steering/blob/main/elections.md#election-schedule-and-operation) should be sufficient. 

To follow the proposed schedule where the new steering committee takes office in December, the election must conclude a week prior, so having the election conclude in November would be ideal to allow more folks to participate, assuming people take time off closer to Dec for the holidays. And, to account for 2-4 weeks of voting as proposed, the election must start in Oct to allow the election to conclude in Nov. So I'm thinking something like this

September
- Election officers appointed
- Election preparation (publish the list of eligible voters, open call for exceptions)

October
- Call for nomination
- Start of election

November
- Conclusion of election
- Results announced within one week after the election concludes

December
- New steering committee members take office

Reply:
I agree with @annajung, but do we want to put specific dates?

Reply:
+1 to Anna's outline, it might be easier for the officials to not define specific dates.

Reply:
+1 to Anna

Reply:
Perhaps we should at least specify December 1st as the date for the new members to take office, for the avoidance of confusion.

Reply:
I added the general timeline proposed by Anna. For the date when new member to take office, I set it to "first week of December" instead of an exact date.